packages/types/src/discordeno.ts

```diff
@@ -1297,3 +1304,25 @@ export interface EditApplication {
    */
   tags?: string[]
 }
+
+/** https://discord.com/developers/docs/resources/poll#poll-create-request-object */
+export interface CreatePoll {
+  /** The question of the poll. Only `text` is supported. */
+  question: Camelize<DiscordPollMedia>
+  /** Each of the answers available in the poll, up to 10 */
+  answers: Array<Omit<Camelize<DiscordPollAnswer>, 'answerId'>>
+  /** Number of hours the poll should be open for, up to 7 days */
+  duration: number
+  /** Whether a user can select multiple answers */
+  allowMultiselect: boolean
+  /** The layout type of the poll */
+  layoutType?: DiscordPollLayoutType
+}
+
+/** https://discord.com/developers/docs/resources/poll#get-answer-voters-query-string-params */
+export interface GetPollAnswerVotes {
+  /** Get users after this user ID */
+  after?: BigString
+  /** Max number of users to return (1-100) */
+  limit?: number
```

Comment:
Should we document [the default to 25](https://discord.com/developers/docs/resources/poll#get-answer-voters-query-string-params) ?

Reply:
idk, might be worth adding it